README.md

```diff
@@ -47,4 +48,31 @@ Make sure that you have installed locally:
     ```
     mvn spring-boot:run
     ```
-3. The server should now be running on http://localhost:8080
+3. The server should now be running on http://localhost:8080  
+
+## Deployment to Google Cloud Platform (GCP)
+
+This application is deployed on **Google Cloud Platform (GCP)** using **Cloud Run**.  
+
+**Live URL:** [GitHub AI Bot](https://github-ai-bot-658818439028.europe-west1.run.app)
+
+### Basic Deployment Steps  
+
```

Comment:
Write a sentence saying that you need to create a project and have the project's ID